logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/SlowSyncWorker.kt

```diff
@@ -23,8 +23,8 @@ class SlowSyncWorker(
             userSessionScope.users.syncSelfUser()
                 .flatMap { userSessionScope.conversations.syncConversations() }
                 .flatMap { userSessionScope.connection.syncConnections() }
-                .flatMap { userSessionScope.team.syncSelfTeamUseCase() }
                 .flatMap { userSessionScope.users.syncContacts() }
+                .flatMap { userSessionScope.team.syncSelfTeamUseCase() }
```

Comment:
Here, I know we have some difficult time discovering `usertypes`, but I would like to have an opinion from someone more expert in this sync topic like @vitorhugods to validate if sides effects could happen, since the sync process has an order documented 😅

Reply:
I will try to revert this change with before current updates may resolved that sync problems

Reply:
reverted

Reply:
I see now that I mentioned and this was already reverted. But indeed, changing this order would break some stuff a bit right now. `SyncContacts` is responsible for fetching information about **all** discovered users, including team members.

Inverting it would make it so we don't fetch details of team members. 

What we could do is change how `SyncContacts` work, and fetch non team members differently from team members inside of it. Which is kind how it's being done after these changes I guess, just with this logic being delegated to the DAO...